### ISSUE TITLE
bs-dropdown menu item actions not firing after 2.1.0

### ISSUE THREAD

**Issue by Techn1x** [kaliber5/ember-bootstrap#660]
Recently upgraded to ember-bootstrap 2.1, but it seems to have broken all of my dropdowns.
When clicking a menu item in the dropdown, the dropdown closes, but action associated with the menu item is no longer called, heres a code snippet:

```handlebars
{{#bs-dropdown as |dd|}}
  {{#dd.button}}
    {{reportModel.reportType}} <span class="caret"></span>
  {{/dd.button}}
  {{#dd.menu renderInPlace=false as |menu|}}
    {{#each reportModel.validReportTypes as |rt|}}
      {{#menu.item class=(if (eq reportModel.reportType rt) ' active' '')}}
        {{!-- reportModel.reportType is never set --}}
        <a class="cursor-pointer" {{action (mut reportModel.reportType) rt}}>{{rt}}</a>
      {{/menu.item}}
    {{/each}}
  {{/dd.menu}}
{{/bs-dropdown}}
```
Adding `onclick={{action ...}}` to the `<a>` element fixes it, but I'm sure my usage without `onclick` attribute should still be supported.

I will try to find some time this week to add a failing test

---

Output from `ember bootstrap:info`
```
Npm packages:ember-bootstrap: ^2.1.0 -> 2.1.0
ember-cli: ~3.2.0 -> 3.2.0bootstrap: n/a
bootstrap-sass: ^3.3.7 -> 3.3.7
ember-cli-sass: ~7.1.5 -> 7.1.7
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
importBootstrapFont: false
bootstrapVersion: 3
importBootstrapCSS: false
importBootstrapTheme: true
```

**Comment by stark-mkl** [comment]
In Addition to this bug, by clicking dropdown a warning message appears about "sendAction" deprecation. It is related to using sendAction on open/close dropdown

`if (this.get('isOpen')) {
                    this.send('closeDropdown');
                  } else {
                    this.send('openDropdown');
}`

`deprecate.js:102 DEPRECATION: You called <x:ember-popper::ember479>.sendAction("registerAPI") but Component#sendAction is deprecated. Please use closure actions instead. [deprecation id: ember-component.send-action] See https://emberjs.com/deprecations/v3.x#toc_ember-component-send-action for more details.`

**Comment by potato20** [comment]
Also, regular ``link-to``'s in a dropdown item are causing full page navigation i.e. reload of the page, instead of regular transitions.

**Comment by simonihmig** [comment]
@stark-mkl this seems unrelated, the deprecation comes from `ember-popper`, and a PR is already prepared: https://github.com/kybishop/ember-popper/pull/92

**Comment by simonihmig** [comment]
A fix is underway: #661 

That thing was pretty weird, have a look there for an (somewhat) explanation.

**Comment by simonihmig** [comment]
Just released the fix as 2.1.1!
